python/src/PointToFieldFunctionalChaosAlgorithm_doc.i.in

```diff
@@ -0,0 +1,273 @@
+%feature("docstring") OT::PointToFieldFunctionalChaosAlgorithm
+"Functional metamodel algorithm based on chaos decomposition.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+The present algorithm allows one to build a response surface of the application :math:`h` of the form:
+
+.. math::
+    h: \left|
+      \begin{array}{ccl}
+         \Rset^p & \rightarrow & \cM_N \times (\Rset^d)^N \\
+         \mat{X} & \mapsto & \vect{Y}
+      \end{array}
+    \right.
+
+The application :math:`h` is known from :math:`n` vector :math:`(\mat{X_1}, \dots, \mat{X_n})`
```

Comment:
What does this mean?

Reply:
The point->field function we want to approximate is known from an input Sample and an output ProcessSample